src/containers/settings/reducers/updateAppSettingsReducer.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { REQUEST_STATE } from '../../../core/redux/constants';
+import {
+  UPDATE_APP_SETTINGS,
+  updateAppSettings,
+} from '../actions';
+
+export default function updateAppSettingsReducer(state :Map, action :SequenceAction) {
+
+  return updateAppSettings.reducer(state, action, {
+    REQUEST: () => state
+      .setIn([UPDATE_APP_SETTINGS, REQUEST_STATE], RequestStates.PENDING)
+      .merge(action.value),
```

Comment:
Why do you merge on REQUEST and not on SUCCESS? Additionally, if the request fails, you are not undoing this change.

Reply:
the merge on request is to avoid issues that might arise if someone were to make many changes faster than the requests would return. in which case the latest state reflected would also be inaccurate if merged on success. we'd have to disable for each request and wait for success to come back before making another change.

ideally there's some error messaging saying some changes may not have gone through. can make those enhancements in another PR